Binding.Mac does not build on Windows

The new Xamarin stuff allows us to build Xamarin.Mac projects on Windows. However, this does not seem to be plug and play and currently Binding.Mac does not build and throws following errors:

```
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSSearchFieldTextTargetBinding.cs(21,10,21,31): warning CS1030: #warning: 'Unlikeley this works!'
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxMacTargetBinding.cs(12,49,12,75): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSViewVisibleTargetBinding.cs(25,25,25,39): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSViewVisibleTargetBinding.cs(30,25,30,31): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxBaseNSDatePickerTargetBinding.cs(61,25,61,39): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSSliderValueTargetBinding.cs(20,50,20,88): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxBaseNSDatePickerTargetBinding.cs(21,19,21,31): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSViewVisibleTargetBinding.cs(15,19,15,25): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSDatePickerDateTargetBinding.cs(34,25,34,29): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSSliderValueTargetBinding.cs(46,25,46,39): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxBaseNSDatePickerTargetBinding.cs(45,53,45,62): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxBaseNSDatePickerTargetBinding.cs(53,48,53,60): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSSliderValueTargetBinding.cs(38,60,38,66): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxBaseNSDatePickerTargetBinding.cs(55,41,55,53): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxBaseNSDatePickerTargetBinding.cs(55,19,55,27): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSDatePickerDateTargetBinding.cs(39,48,39,60): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSViewVisibleTargetBinding.cs(30,37,30,47): error CS1715: 'MvxNSViewVisibleTargetBinding.TargetType': type must be 'Type' to match overridden member 'MvxTargetBinding.TargetType'
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSViewVisibleTargetBinding.cs(30,37,30,47): error CS0012: The type 'Type' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSViewVisibleTargetBinding.cs(13,18,13,47): error CS0534: 'MvxNSViewVisibleTargetBinding' does not implement inherited abstract member 'MvxTargetBinding.TargetType.get'
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSDatePickerDateTargetBinding.cs(34,30,34,40): error CS1715: 'MvxNSDatePickerDateTargetBinding.TargetType': type must be 'Type' to match overridden member 'MvxTargetBinding.TargetType'
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSDatePickerDateTargetBinding.cs(34,30,34,40): error CS0012: The type 'Type' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSDatePickerDateTargetBinding.cs(16,18,16,50): error CS0534: 'MvxNSDatePickerDateTargetBinding' does not implement inherited abstract member 'MvxTargetBinding.TargetType.get'
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSViewVisibleTargetBinding.cs(20,46,20,52): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSSliderValueTargetBinding.cs(22,61,22,73): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxBaseNSDatePickerTargetBinding.cs(26,52,26,64): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSDatePickerDateTargetBinding.cs(18,49,18,61): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSTextViewTextTargetBinding.cs(20,51,20,91): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSDatePickerTimeTargetBinding.cs(51,25,51,29): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSSearchFieldTextTargetBinding.cs(23,54,23,97): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSTextViewTextTargetBinding.cs(46,25,46,39): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSButtonTitleTargetBinding.cs(31,25,31,39): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSButtonTitleTargetBinding.cs(36,25,36,31): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSSliderValueTargetBinding.cs(37,10,37,16): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSSliderValueTargetBinding.cs(37,10,37,34): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSTextViewTextTargetBinding.cs(38,55,38,64): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSDatePickerTimeTargetBinding.cs(40,48,40,60): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSButtonTitleTargetBinding.cs(17,19,17,27): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSSearchFieldTextTargetBinding.cs(46,25,46,39): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSDatePickerTimeTargetBinding.cs(51,30,51,40): error CS1715: 'MvxNSDatePickerTimeTargetBinding.TargetType': type must be 'Type' to match overridden member 'MvxTargetBinding.TargetType'
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSDatePickerTimeTargetBinding.cs(51,30,51,40): error CS0012: The type 'Type' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSDatePickerTimeTargetBinding.cs(16,18,16,50): error CS0534: 'MvxNSDatePickerTimeTargetBinding' does not implement inherited abstract member 'MvxTargetBinding.TargetType.get'
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSSwitchOnTargetBinding.cs(20,47,20,85): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSButtonTitleTargetBinding.cs(36,37,36,47): error CS1715: 'MvxNSButtonTitleTargetBinding.TargetType': type must be 'Type' to match overridden member 'MvxTargetBinding.TargetType'
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSButtonTitleTargetBinding.cs(36,37,36,47): error CS0012: The type 'Type' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSButtonTitleTargetBinding.cs(15,18,15,47): error CS0534: 'MvxNSButtonTitleTargetBinding' does not implement inherited abstract member 'MvxTargetBinding.TargetType.get'
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSDatePickerTimeTargetBinding.cs(18,49,18,61): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSTextViewTextTargetBinding.cs(22,62,22,74): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSButtonTitleTargetBinding.cs(22,46,22,54): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSSwitchOnTargetBinding.cs(60,25,60,39): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSSearchFieldTextTargetBinding.cs(25,65,25,77): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSSwitchOnTargetBinding.cs(36,58,36,64): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSSegmentedControlSelectedSegmentTargetBinding.cs(13,70,13,118): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSSearchFieldTextTargetBinding.cs(40,10,40,16): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSSearchFieldTextTargetBinding.cs(40,10,40,38): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSSwitchOnTargetBinding.cs(22,58,22,70): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSSegmentedControlSelectedSegmentTargetBinding.cs(30,25,30,39): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSSegmentedControlSelectedSegmentTargetBinding.cs(22,56,22,62): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSSegmentedControlSelectedSegmentTargetBinding.cs(17,81,17,93): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Views\MvxTableViewSource.cs(42,10,42,34): error CS0012: The type 'Attribute' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSTextFieldTextTargetBinding.cs(21,52,21,93): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSViewVisibilityTargetBinding.cs(28,25,28,39): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSViewVisibilityTargetBinding.cs(33,25,33,31): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSTextFieldTextTargetBinding.cs(46,25,46,39): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSViewVisibilityTargetBinding.cs(18,19,18,25): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\MvxMacBindingBuilder.cs(21,11,21,28): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSTextFieldTextTargetBinding.cs(38,59,38,68): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSViewVisibilityTargetBinding.cs(33,37,33,47): error CS1715: 'MvxNSViewVisibilityTargetBinding.TargetType': type must be 'Type' to match overridden member 'MvxTargetBinding.TargetType'
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSViewVisibilityTargetBinding.cs(33,37,33,47): error CS0012: The type 'Type' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSViewVisibilityTargetBinding.cs(16,18,16,50): error CS0534: 'MvxNSViewVisibilityTargetBinding' does not implement inherited abstract member 'MvxTargetBinding.TargetType.get'
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSTextFieldTextTargetBinding.cs(23,63,23,75): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\Target\MvxNSViewVisibilityTargetBinding.cs(23,49,23,55): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\MvxMacBindingBuilder.cs(36,53,36,85): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\MvxMacBindingBuilder.cs(83,67,83,99): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\MvxMacBindingBuilder.cs(84,67,84,71): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\MvxMacBindingBuilder.cs(84,85,84,89): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\MvxMacBindingBuilder.cs(89,53,89,79): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\MvxMacBindingBuilder.cs(97,57,97,80): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\MvxMacBindingBuilder.cs(23,26,23,66): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\MvxMacBindingBuilder.cs(23,33,23,65): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\MvxMacBindingBuilder.cs(24,26,24,60): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\MvxMacBindingBuilder.cs(24,33,24,59): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\MvxMacBindingBuilder.cs(25,26,25,57): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\MvxMacBindingBuilder.cs(25,33,25,56): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\MvxMacBindingBuilder.cs(27,37,27,77): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\MvxMacBindingBuilder.cs(27,44,27,76): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\MvxMacBindingBuilder.cs(28,37,28,71): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\MvxMacBindingBuilder.cs(28,44,28,70): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\MvxMacBindingBuilder.cs(29,37,29,68): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
1>D:\git\MvvmCross\MvvmCross\Binding\Mac\MvxMacBindingBuilder.cs(29,44,29,67): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
```

I have tried adding the `System.Runtime` nuget, which does not change anything.

--- a/MvvmCross/Binding/Mac/MvvmCross.Binding.Mac.csproj
+++ b/MvvmCross/Binding/Mac/MvvmCross.Binding.Mac.csproj
@@ -74,7 +74,7 @@
     <Compile Include="Target\MvxNSSegmentedControlSelectedSegmentTargetBinding.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
   </ItemGroup>
-  <Import Project="$(MSBuildBinPath)\Microsoft.CSharp.targets" />
+  <Import Project="$(MSBuildExtensionsPath)\Xamarin\Mac\Xamarin.Mac.CSharp.targets" />
   <ItemGroup>
     <Folder Include="Properties\" />
   </ItemGroup>

--- a/MvvmCross/Mac/Mac/MvvmCross.Mac.csproj
+++ b/MvvmCross/Mac/Mac/MvvmCross.Mac.csproj
@@ -77,7 +77,7 @@
     <Compile Include="Properties\AssemblyInfo.cs" />
     <Compile Include="ExtensionMethods\MvxMacDateTimeExtensionMethods.cs" />
   </ItemGroup>
-  <Import Project="$(MSBuildBinPath)\Microsoft.CSharp.targets" />
+  <Import Project="$(MSBuildExtensionsPath)\Xamarin\Mac\Xamarin.Mac.CSharp.targets" />
   <ItemGroup>
     <ProjectReference Include="..\..\Binding\Mac\MvvmCross.Binding.Mac.csproj">
       <Project>{F089E103-7F5D-4000-A4B0-B1D9F37594D9}</Project>

--- a/MvvmCross/Platform/Mac/MvvmCross.Platform.Mac.csproj
+++ b/MvvmCross/Platform/Mac/MvvmCross.Platform.Mac.csproj
@@ -58,7 +58,7 @@
     <Compile Include="Views\IMvxEventSourceViewController.cs" />
     <Compile Include="Views\MvxBaseViewControllerAdapter.cs" />
   </ItemGroup>
-  <Import Project="$(MSBuildBinPath)\Microsoft.CSharp.targets" />
+  <Import Project="$(MSBuildExtensionsPath)\Xamarin\Mac\Xamarin.Mac.CSharp.targets" />
   <ItemGroup>
     <Folder Include="Platform\" />
   </ItemGroup>

